src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -174,8 +174,10 @@ var DOMPropertyOperations = {
         var propName = propertyInfo.propertyName;
         // Must explicitly cast values for HAS_SIDE_EFFECTS-properties to the
         // property type before comparing; only `value` does and is string.
+        // Must set `value` property if it is not null and not yet set.
```

Comment:
You call out `value` as being special here but this will apply to other things too (eg  `selected` which is also `MUST_USE_PROPERTY`). Is that your intention?